fish_prompt.fish

```
@@ -895,16 +896,30 @@ function __bobthefish_prompt_desk -S -d 'Display current desk environment'
     set_color normal
 end
 
-function __bobthefish_prompt_nvm -S -d 'Display current node version through NVM'
-    [ "$theme_display_nvm" = 'yes' -a -n "$NVM_DIR" ]
+function __bobthefish_prompt_node -S -d 'Display current node version'
+    [ "$theme_display_node" = "yes" -o "$theme_display_nvm" = 'yes' ]
     or return
 
-    set -l node_version (nvm current 2> /dev/null)
+    set -l node_manager
+    set -l node_manager_dir
+
+    if type -fq nvm
+      set node_manager "nvm"
+      set node_manager_dir $NVM_DIR
+    else if type -fq fnm
+      set node_manager "fnm"
+      set node_manager_dir $FNM_DIR
+    end
+
+    [ -n "$node_manager_dir" ]
+    or return
+
+    set -l node_version ("$node_manager" current)
```

we should swallow errors here, as before (redirect stderr to /dev/null)